internal/authtoken/repository.go

```diff
@@ -45,15 +49,27 @@ func NewRepository(r db.Reader, w db.Writer, kms *kms.Kms, opt ...Option) (*Repo
 	}
 
 	opts := getOpts(opt...)
+
 	if opts.withLimit == 0 {
 		// zero signals the boundary defaults should be used.
 		opts.withLimit = db.DefaultLimit
 	}
+
+	if opts.withTokenTimeToLiveDuration == 0 {
```

Comment:
Don't do this here. Have the `getDefaultOptions` function do what it's meant to do, as we do in other packages.